### PR TITLE
Add testcase for PYACTION generated UDQ

### PR DESCRIPTION
**Pull request by joakim-hove**
In this testcase the UDQ values are evaluated in Python instead of the .DATA deck. Case is identical to existing UDQ_WCONPROD case. 

Not everything to actually run this is merged yet - but it is quite close

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Can this be merged?

File: udq_actionx/PYACTION_WCONPROD.DATA [hunk 1]
```diff
@@ -0,0 +1,409 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2018 Equinor
+
+-- This deck uses User defined quantities (UDQ) to calculated
+-- production constrints which are set with keyword WCONPROD.
```
Comment (akva2) [inline udq_actionx/PYACTION_WCONPROD.DATA:9]: constrAints

**Comment by akva2** [comment]
apart from the typo this is improvement and additions so i reckon merge at will.

**Comment by joakim-hove** [comment]
Typo fixed - merging

**Pull request merged**